### DISCUSSION TITLE
Compilation error, libiconv not found

### DISCUSSION THREAD

**Discussion by fa2002** [filecoin-project/lotus#5573] [Storage Provider]
Excuse me, why do I always encounter this error in compiling. Actually, I also have the libiconv file.

git clone https://github.com/filecoin-project/lotus.git
cd /lotus/
make clean all
![image](https://user-images.githubusercontent.com/631480/107487009-aa4d0800-6bc0-11eb-87de-ff5c3347a3c7.png)

**Comment by rjan90** [comment]
Have you installed all the dependencies? [https://docs.filecoin.io/get-started/lotus/installation/#software-dependencies](https://docs.filecoin.io/get-started/lotus/installation/#software-dependencies)

**Reply by fa2002** [reply]
I installed the required extensions——GO、RUST and "yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm; sudo yum install -y git gcc bzr jq pkgconfig clang llvm mesa-libGL-devel opencl-headers ocl-icd ocl-icd-devel hwloc-devel"  and other things I can think of.
Libiconv also exists locally——
![image](https://user-images.githubusercontent.com/631480/107530561-f6b33a80-6bf6-11eb-9eda-cf02970dabb2.png)

But why still can't find libiconv？
